modules/new_serverpod_auth/serverpod_auth_jwt/serverpod_auth_jwt_server/lib/src/generated/refresh_token.dart

```diff
@@ -0,0 +1,604 @@
+/* AUTOMATICALLY GENERATED CODE DO NOT MODIFY */
+/*   To generate run: "serverpod generate"    */
+
+// ignore_for_file: implementation_imports
+// ignore_for_file: library_private_types_in_public_api
+// ignore_for_file: non_constant_identifier_names
+// ignore_for_file: public_member_api_docs
+// ignore_for_file: type_literal_in_constant_pattern
+// ignore_for_file: use_super_parameters
+
+// ignore_for_file: unnecessary_null_comparison
+
+// ignore_for_file: no_leading_underscores_for_library_prefixes
+import 'package:serverpod/serverpod.dart' as _i1;
+import 'package:serverpod_auth_user_server/serverpod_auth_user_server.dart'
+    as _i2;
+import 'package:serverpod_auth_jwt_server/src/generated/protocol.dart' as _i3;
+
+abstract class RefreshToken
+    implements _i1.TableRow<_i1.UuidValue?>, _i1.ProtocolSerialization {
+  RefreshToken._({
+    this.id,
+    required this.authUserId,
+    this.authUser,
+    required this.scopeNames,
+    required this.fixedSecret,
+    required this.variableSecret,
+    DateTime? lastUpdated,
+    DateTime? created,
+  })  : lastUpdated = lastUpdated ?? DateTime.now(),
+        created = created ?? DateTime.now();
+
+  factory RefreshToken({
+    _i1.UuidValue? id,
+    required _i1.UuidValue authUserId,
+    _i2.AuthUser? authUser,
+    required Set<String> scopeNames,
+    required String fixedSecret,
+    required (String, String) variableSecret,
```

Comment:
I should fix the code generation to re-expose the names of the positional fields (for documentation purposes).

Then we can retain the small storage and don't have to use a full named record or have to resort to `String`-building ourselves.